reana_cluster/backends/kubernetes/templates/deployments/server-template.yaml

```diff
@@ -57,6 +57,8 @@ spec:
             secretKeyRef:
               name: reana-db-secrets
               key: password
+        - name: REANA_SQLALCHEMY_DATABASE_URI
```

Comment:
Why did you need to do this? Normally DB details are passed [here](https://github.com/reanahub/reana-cluster/blob/7b24a3a700fb8027dc1157b853b9c0eea400f4d7/reana_cluster/configurations/reana-cluster-minikube.yaml#L26) :)

Reply:
Oops, I apologize, we had an old version of reana.db, where there SQLALCHEMY_DATABASE_URI was not generated atomatically from credentials, like it is now [here](https://github.com/reanahub/reana-db/blob/master/reana_db/config.py#L32)
So with new version of reana.db code it is really unnecessary.